js/main.js

```diff
@@ -0,0 +1,89 @@
+///////////////////////////////////////////////////////////////
+///////////   ORIGINAL CODE ////////////////////
+///////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////
+// var entries = $(".entries");
+//
+// var form = $("form");
+// var inputBox = $(".newEntry");
+//
+// var totalEl = $(".total");
+// var total = 0.00;
+//
+// form.on("submit", updateRegister);
+//
+// function updateRegister() {
+//   event.preventDefault();
+//
+//   // get the amount and clear the box
+//   var amount = parseFloat(inputBox.val());
+//   inputBox.val("");
+//
+//   // make the new element and add it to the list of entries
+//   var newEl = $("<tr>");
+//   newEl.append("<td></td>");
+//   newEl.append("<td>" + "$" + amount.toFixed(2) + "</td>");
+//
+//   entries.append(newEl);
+//
+//   // update the total
+//   total = total + amount;
+//   totalEl.text("$" + total.toFixed(2));
+// }
+///////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////
+function Register(){
+    this.domElement = null;
+    this.total = 0.00;
+
+    this.initialize = function(container){
+        this.domElement = $('<div class="register"></div>');
```

Comment:
a convention we like to follow as developers is `el` for the element that the component's primary focus will be, couple letters shorter is all, but you might save you some time moving forward.